test/functional/ios/helper/test_helper.py

```diff
@@ -28,3 +29,14 @@ def setUp(self):
 
     def tearDown(self):
         self.driver.quit()
+
+
+def get_xcode_ver():
+    """Returns Xcode version
+
+    Returns:
+        str: Xcode version (e.g. '11.3')
+    """
+    lines = sp.check_output(['xcodebuild', '-version'])
+    return [line.decode('utf-8').split()[-1] for line in lines.strip().split(b'\n')
```

Comment:
You can handle here like below.

```python
match = re.match(r'Xcode ([0-9\.]+)+', lines)
return None if match is None else match.group(1)
```

How to handle here if the index error occurred? Will called test case will fail because of index error?

Reply:
Thanks for your comments.
I've removed here for now since it's unnecessary. (but I'll use `re` if possible for future.)